docs/introduction/install.rst

```diff
@@ -2,67 +2,54 @@
 Install Auspice
 ===============
 
-Prerequisites
-=============
+.. note::
+   This is an Auspice-specific installation guide. If you wish to use Nextstrain as a whole, please refer to `the Nextstrain installation guide <https://docs.nextstrain.org/en/latest/install.html>`__.
 
-Auspice is a JavaScript program, and requires `Node.js <https://nodejs.org/>`__ to be installed on your system. Refer to ``engines.node`` in `package.json <https://github.com/nextstrain/auspice/blob/-/package.json>`__ for currently supported versions.
+.. contents::
+   :local:
 
-We highly recommend using `Conda <https://conda.io/docs/>`__ to manage environments, i.e. use Conda to create an environment with Node.js installed where you can use Auspice. It's possible to use other methods, but this documentation presupposes that you have Conda installed.
+Install dependencies
+====================
 
-To run package scripts, the `bash shell <https://en.wikipedia.org/wiki/Bash_(Unix_shell)>`__ and the `env <https://en.wikipedia.org/wiki/Env>`__ command need to be in your ``PATH``. You should already have them on Unix-like systems including Linux and macOS. If you are working from Windows, you can run the installation under Git Bash, MSYS2, or Cygwin. You can also use the Windows Subsystem Linux for a fuller Linux environment.
+Auspice is a JavaScript program, and requires `Node.js <https://nodejs.org/>`__ to be installed on your system. Refer to ``engines.node`` in `package.json <https://github.com/nextstrain/auspice/blob/-/package.json>`__ for currently supported versions.
 
-Create a Conda Environment
-==========================
+We recommend using `Mamba <https://mamba.readthedocs.io/>`__ to create an environment with a specific version of Node.js. It's possible to use other methods, but these are the instructions for Mamba:
 
 .. code:: bash
 
-   conda create --name auspice nodejs=14
+   mamba create -c conda-forge --name auspice nodejs=14
```

Comment:
Should we update this to nodejs=16 yet?

Reply:
For what it's worth, `conda` is fast for a simple dependency chain such as this and is what I still use.

Reply:
> For what it's worth, `conda` is fast for a simple dependency chain such as this and is what I still use.

You're right, I just tested and even after `conda clean --all --yes` it's still reasonably fast. I'll revert.

Reply:
Conda reference reverted in [force-push](https://github.com/nextstrain/auspice/compare/5c0f03aa0a8ea792c10bfee4392aca7db87b5bc7..411324b085ac9b6e61aa02cdb082362073d45f95) and Node.js version bump added in 5cfccafbdd2e5c7b4bc36f705ffaec2835e5d07c.